controller/server/in_mem_service_storage_test.go

```diff
@@ -261,7 +261,7 @@ func TestDeleteBrokerMultiple(t *testing.T) {
 		t.Fatalf("Failed to get inventory: %#v", err)
 	}
 	if len(catRet.Services) != 2 {
-		t.Fatalf("Expected 2 services from GetInventory, got %s ", len(catRet.Services))
+		t.Fatalf("Expected 2 services from GetInventory, got %d ", len(catRet.Services))
```

Comment:
It might be nice if it were: `... got %#v", catRetServices)`  so that we can see the number and values returned.
(if I have the % syntax right)